devicetypes/smartthings/smartweather-station-tile.src/smartweather-station-tile.groovy

```diff
@@ -242,8 +254,8 @@ def pollUsingZipCode(String zipCode) {
 
         send(name: "humidity", value: obs.relativeHumidity, unit: "%")
         send(name: "weather", value: obs.wxPhraseShort)
-        send(name: "weatherIcon", value: obs.iconCode as String, displayed: false)
-        send(name: "wind", value: obs.windSpeed as String, unit: windUnits) // as String because of bug in determining state change of 0 numbers
```

Comment:
Is the issue described in the comment still relevant?

Reply:
I haven't experienced that.